index.js

```diff
@@ -15,7 +16,7 @@ class MyServer {
 
         return new Promise((resolve, reject) => {
             this.server = app.listen(portNum, () => resolve()).on('error', reject);
-            console.log(`Listening on port ${portNum}`);
+            debug(`Listening on port ${portNum}`);
```

Comment:
It will be more correct if we output this message right before calling `resolve()`, don't you think? Also, it is very important to log any error in your app, so it will be wonderful if you log errors in all 'error` handlers.

Reply:
Yeah, logging right before resolve() makes more sense.
Working on it now.